src/main/webapp/js/utils/form/form-utils.jsx

```diff
@@ -115,15 +115,22 @@ const renderFormField = (props) => {
   );
 };
 
-export const validate = fieldConfig => (values) => {
+export const validate = (fieldConfig, fieldConfigs) => (values) => {
   const errors = {};
-
   _.forEach(fieldConfig, (config) => {
     const val = values[config.name];
 
     if (config.required && (_.isNil(val) || val === '')) {
       errors[config.name] = `${config.displayName} is required`;
     }
+    if (config.uniqueField && !(_.isNil(val) || val === '')) {
+      const uniqueEntity = _.find(
```

Comment:
it's not unique entity, it's more like existing entity or duplicated entity. I think the most descriptive will be entityWithDuplicatedField, but it's a little long